jobrunner/sync.py

```diff
@@ -104,6 +104,11 @@ def job_to_remote_format(job):
         ]
     }
 
+    # convert cross-system identifier key
+    data["runner_id"] = job.id
```

Comment:
I think it would be worth removing `id` from the list of keys above to avoid any confusion.